readthedocs/settings/base.py

```diff
@@ -59,6 +59,8 @@ class CommunityBaseSettings(Settings):
     SESSION_COOKIE_DOMAIN = 'readthedocs.org'
     SESSION_COOKIE_HTTPONLY = True
     CSRF_COOKIE_HTTPONLY = True
+    # See: docs/advertising-details.rst
+    CSRF_COOKIE_AGE = None  # session cookie (expires on browser quit)
```

Comment:
This will break user experience

Reply:
How so?

Reply:
This is potentially true and is warned about [in the Django docs](https://docs.djangoproject.com/en/2.0/ref/settings/#csrf-cookie-age). I don't think it will be a big issue for us since it only affects form submissions of which the only ones on a non-authed page are the login/signup forms. However, we could make the CSRF cookie age match the logged in cookie age (~2 weeks) to mitigate it. Thoughts?

Reply:
I set it to 30 days so it matches the GA cookie. I think that's a reasonable balance so it's pretty obvious we aren't using it to track users.